chia/daemon/server.py

```diff
@@ -396,6 +396,8 @@ def _build_plotting_command_args(self, request: Any, ignoreCount: bool) -> List[
         b = request["b"]
         u = request["u"]
         r = request["r"]
+        f = request["f"]
+        p = request["p"]
```

Comment:
I think these need a .get here actually

Reply:
Looking how '-a' option is treated, yes.